.travis/install-micropython.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -ex
+
+cd ~/
+
+# Build micropython from source
+# TODO: cache micropython build output
```

Comment:
Here is how I did it:

Adding a path to travis cache:

https://github.com/ddemidov/amgcl/blob/master/.travis.yml#L39-L42

Checking if path exists (that is, if cache was unpacked successfully):

https://github.com/ddemidov/amgcl/blob/master/.travis/install_boost.sh#L5